tasks/install_shell.sh

```diff
@@ -419,8 +417,7 @@ install_file() {
       info "installing puppetlabs dmg with hdiutil and installer"
       mountpoint="$(mktemp -d -t $(random_hexdump))"
       /usr/bin/hdiutil attach "${download_filename?}" -nobrowse -readonly -mountpoint "${mountpoint?}"
-      latest_macos_version $mountpoint
-      /usr/sbin/installer -pkg "${mountpoint?}/puppet-agent-${dmg_version?}-installer.pkg" -target /
+      /usr/sbin/installer -pkg ${mountpoint?}/puppet-agent-*-installer.pkg -target /
```

Comment:
Using the wildcard to match version in .pkg path is brilliant. Wish I would have thought of that!

Reply:
This change is only needed for PC1 packages. But it seems reasonable enough.